cg/store/crud/read.py

```diff
@@ -1728,17 +1728,20 @@ def get_all_pools_to_deliver(self) -> list[Pool]:
         )
         return records.all()
 
-    def get_orders(self, orders_request: OrdersRequest) -> list[Order]:
-        filters: list[OrderFilter] = [
-            OrderFilter.BY_WORKFLOW,
-            OrderFilter.APPLY_LIMIT,
-        ]
-        return apply_order_filters(
+    def get_orders(self, orders_request: OrdersRequest) -> tuple[list[Order], int]:
```

Comment:
the previous functionality not required anymore, i.e. with the limit?

Reply:
No!